src/sage/libs/pari/convert_sage_matrix.pyx

```diff
@@ -0,0 +1,105 @@
+# sage_setup: distribution = sagemath-pari
```

Comment:
What is this file?

Reply:
slipped in while cherry picking, I'll fix it